schema.sql

```diff
@@ -13,3 +13,24 @@ weight_kg decimal,
 PRIMARY KEY(id)
 );
 ALTER TABLE animals ADD COLUMN species VARCHAR(50);
+
+CREATE TABLE owners (
+id INT GENERATED ALWAYS AS IDENTITY,
+full_name VARCHAR(100),
+age INT,
+PRIMARY KEY(id)
+);
+
+CREATE TABLE species (
+id INT GENERATED ALWAYS AS IDENTITY,
+name VARCHAR(100),
+PRIMARY KEY (id)
+);
+
+ALTER TABLE animals DROP COLUMN species;
+
+ALTER TABLE animals ADD COLUMN species_id INT; 
+ALTER TABLE animals ADD FOREIGN KEY (species_id) REFERENCES species (id);
+
+ALTER TABLE animals ADD COLUMN owner_id INT;
+ALTER TABLE animals ADD FOREIGN KEY (owner_id) REFERENCES owners (id);
```

Comment:
[OPTIONAL]

- Awesome approach to altering the animals table. 👏 
What do you think about adding these columns and also making them foreign keys in two lines of code(one for each column)? Shorter, right? 😀 

You can do it like this 👇 

```
ALTER TABLE animals ADD species_id int REFERENCES species(id);

ALTER TABLE animals ADD owner_id int REFERENCES owners(id);
```

Reply:
Wow, I didn't think of that, an awesome approach. Thank you.

Reply:
You're welcome, @Lockless404 😀